charts/flyte-sandbox/templates/proxy/configmap.yaml

```diff
@@ -94,6 +94,13 @@ data:
                                               route:
                                                   cluster: flyte_grpc
                                             - match:
+                                                  prefix: "/flyteidl.service.SignalService"
+                                              route:
+                                                  cluster: flyte_grpc
+                                            - match:
```

Comment:
Is this a rogue block?

Reply:
oops, i resolved merge conflict incorrectly